src/styles/style.css

```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,700&display=swap");
+ 
+
+.card_page {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 0 16px;
+}
+
+.container__img {
+  align-self: center;
+  margin-top: 32px;
+  min-height: 134px;
+  max-width: 160px;
+  margin-bottom: 40px;
+}
```

Comment:
```suggestion
.container__img {
  width: 160px;
  height: 134px;
  margin-top: 32px auto 40px;
}
```

Reply:
I set this margin and my code didn't  work properly